docs/quickstart/cross-chain-transfers.md

```diff
@@ -0,0 +1,480 @@
+
+:::
+
+<br></br>
+
+Select the **`examples/avm`** folder to view the AvalancheJS X-Chain examples. To send AVAX from the X-Chain to the C-Chain, select [`avm/buildExportTx-cchain-avax.ts`](https://github.com/ava-labs/avalanchejs/blob/master/examples/avm/buildExportTx-cchain-avax.ts)
+
+![Image for post](/img/ajs-export-cchain-res.png)
+
+### Modify your Avalanche network configuration
+
+Depending on how you started your node, you might use custom settings for your Avalanche Network
+
+You might notice that Avalanche Network Runner produces new ports for each node during node cluster deployment. Be sure to change configurations accordingly with each new cluster.
+
+_Note: See [Advanced Configurations](https://docs.avax.network/nodes/build/set-up-node-with-installer#advanced-node-configuration) to learn more about custom Node configurations_
+
+Below are the following objects in your Avalanche Network:
+
+```js
+const ip: string = "localhost";
+const port: number = 9650;
+const protocol: string = "http";
+const networkID: number = 1337;
+```
+
+**IP Address**
+
+You Node's IP address
+
+By default, Avalanche Network Runner uses localhost.
+
+```js
+const ip: string = "localhost";
+```
+
+**Port Number**
+
+As stated above, Avalanche Network Runner produces new port numbers for each node cluster.
+
+```js
+const port: number = "<YOUR-NODE-PORT-HERE>";
+```
+
+**Protocol**
+
+The protocol we use to connect to the node's IP
+
+```js
+const protocol: string = "http";
+```
+
+#### **Network ID**
+
+Depending on the networkID which is passed in when instantiating Avalanche, the encoded addresses used will have a distinctive Human Readable Part(HRP) per each network.
+
+_Example Address: 1337 - X-`custom`19rknw8l0grnfunjrzwxlxync6zrlu33yeg5dya_
+
+Mainnet:
+
+```js
+const networkID: number = 1;
+```
+
+Fuji Testnet:
+
+```js
+const networkID: number = 5;
+```
+
+Custom Network:
+
+```js
+const networkID: number = 1337;
+```
+
+We use Bech32 to encode our addresses. Each Bech32 address is composed of the following components:
+
+- A Human-Readable Part (HRP).
+- The number “1” is a separator (the last digit 1 seen is considered the separator).
+- Base-32 encoded string for the data part of the address (the 20-byte address itself).
+- A 6-character base-32 encoded error correction code using the BCH algorithm.
+
+AvalancheJS has address encoding for the following networks
+
+```text
+0 - X-custom19rknw8l0grnfunjrzwxlxync6zrlu33yeg5dya
+1 - X-avax19rknw8l0grnfunjrzwxlxync6zrlu33y2jxhrg
+2 - X-cascade19rknw8l0grnfunjrzwxlxync6zrlu33ypmtvnh
+3 - X-denali19rknw8l0grnfunjrzwxlxync6zrlu33yhc357h
+4 - X-everest19rknw8l0grnfunjrzwxlxync6zrlu33yn44wty
+5 - X-fuji19rknw8l0grnfunjrzwxlxync6zrlu33yxqzg0h
+1337 - X-custom19rknw8l0grnfunjrzwxlxync6zrlu33yeg5dya
+12345 - X-local19rknw8l0grnfunjrzwxlxync6zrlu33ynpm3qq
+```
```

Comment:
Is this section needed? I feel this just introduces more complexity. Maybe move this to an appendix section.